Components/Core/IconDataProcessors.lua

```diff
@@ -752,6 +752,17 @@ do
 		example = '[MaxDuration] => "3"; [MaxDuration:TMWFormatDuration] => "3.0"; [MaxDuration(icon="TMW:icon:1I7MnrXDCz8T")] => "60"',
 		category = L["ICON"],
 	})
+
+	TMW.C.IconComponent:RegisterDogTag("TMW", "KeystoneLevel", {
```

Comment:
This doesn't really belong here. I'd say that this DogTag doesn't really belong in TMW at all, but rather in LDT itself.

If you open a separate issue on https://www.wowace.com/projects/libdogtag-3-0/issues and then link the issue you create to me, I can add this to LDT.